pkg/plugin/utils.go

```diff
@@ -36,33 +36,38 @@ const (
 )
 
 func GetIpsByOnField(on string, context Context, phase Phase) (ipList []net.IP, err error) {
-	on = strings.TrimSpace(on)
-	if strings.Contains(on, EqualSymbol) {
-		if (phase != PhasePostInstall && phase != PhasePostJoin) && phase != PhasePreClean {
-			logrus.Warnf("Current phase is %s. When nodes is specified with a label, the plugin action must be PostInstall or PostJoin, ", phase)
-			return nil, nil
+	splits := strings.Split(on, SplitSymbol)
+	for _, split := range splits {
+		var ips []net.IP
+		split = strings.TrimSpace(split)
+		if strings.Contains(split, EqualSymbol) {
+			if (phase != PhasePostInstall && phase != PhasePostJoin) && phase != PhasePreClean {
+				logrus.Warnf("Current phase is %s. When nodes is specified with a label, the plugin action must be PostInstall or PostJoin, ", phase)
+				return nil, nil
```

Comment:
I think we should return the detailed error, instead of returning nil. What do you think? @YTGhost

Reply:
You're right. It'll be better, Thx